src/app/onion/learning-object-builder/components/info-page/metadata/metadata.component.ts

```diff
@@ -30,4 +65,52 @@ export class LearningObjectMetadataComponent implements OnInit {
       this.learningObject.addLevel(level);
     }
   }
+
+  search() {
+    this.query.text = this.query.text.trim();
+      this.userService.searchUsers(this.query.text).then(val => {
+        this.users = JSON.parse(val);
```

Comment:
Won't have to parse once return type is `User[]` after using `User.instantiate()` on payload elements